cvefeed/nvd/match_cpe.go

```diff
@@ -114,6 +114,10 @@ func (cm *cpeMatch) match(attr *wfn.Attributes, requireVersion bool) bool {
 		return false
 	}
 
+	if cm.hasVersionRanges && attr.Version == wfn.NA {
```

Comment:
Thanks for contributing. Overall, LGTM, but `cm.hasVersionRanges` check is redundant, it's already been checked above.

Reply:
@skogtwin how is it going？

Reply:
Hi @yywing. I believe the PR is ready to merge, we are waiting for you to sign CLA.

Reply:
done. wait for 1h